scripts/e2e-docker-start

```diff
@@ -21,12 +21,18 @@ fi
 
 echo "Using Rancher image version: ${RANCHER_IMG_VERSION}"
 
+# Configure the Rancher server URL from start, similarly to how chart installations work
+# The special host.docker.internal domain only works for Docker Desktop, not plain Docker Engine on linux, so using the special IP 172.17.0.1 to access the gateway of the default bridge network
+SERVER_URL_ARGS="-e CATTLE_SERVER_URL=https://172.17.0.1"
```

Comment:
Can this just be `SERVER_URL="https://172.17.0.1"`, just to follow file convention